middleware/request_logs.rb

```diff
@@ -9,10 +9,12 @@ def initialize(app, request_logs)
       def call(env)
         request_id = env['cf.request_id']
         @request_logs.start_request(request_id, env)
+        start_time = Time.now
 
         status, headers, body = @app.call(env)
-
-        @request_logs.complete_request(request_id, status)
+        # convert to milliseconds
+        time_taken = (Time.now - start_time) * 1000
```

Comment:
Maybe also convert to an integer (`.to_i`)? We don't need fractions of milliseconds I guess.